.github/workflows/release.yml

```
@@ -34,3 +36,14 @@ jobs:
           asset_path: normalized
           asset_name: package-set.dhall
           asset_content_type: application/x-dhall
+      - name: Generate Package Listing
+        run: |
+          cd generate-package-list
+          npm install --only=production
+          npm run build ${{ github.ref }}
+      - name: Upload Package Listing
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages
+          FOLDER: generate-package-list/dist
```

I like
```suggestion
          FOLDER: generate-package-list/dist
          SINGLE_COMMIT: yes
```
else the `gh-pages` branch just keeps increasing the size of the repo.

---

Sounds good, thanks!